test/logging_unittest.cc

```diff
@@ -35,7 +35,7 @@ TEST(LoggingTest, Logger) {
 
   EXPECT_TRUE(boost::regex_match(out.str(), boost::regex(
      "W\\d{4} \\d{2}:\\d{2}:\\d{2} .+ "
```

Comment:
Can you update the ".+" on this line to be more strict? \d+ should be fine

Reply:
Done